index2.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="projects.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>works section</title>
+</head>
+<body>
+  <section class="works"> 
+  <div class="parent-container">
+    <div class="snapshot"><img src="snapshotimages/Snapshoot-Portfolio.png" alt="snapshot"></div>
+    <div class="individual-card">
+      <div><h2>Tonic</h2></div>
+      <nav class="nav-elements">
+        <div><a class="current-nav" href="http://">Canopy</a></div>
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">Back End Dev</a></div>
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">2015</a></div>
+      </nav>
+      <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+      <ul class="languages">
+        <li><button class="language-button">html</button></li>
+        <li><button class="language-button">css</button></li>
+        <li><button class="language-button">javascript</button></li>
+      </ul>
+      <div><button class="see-project-button">See Project</button></div>
+    </div>
+  </div>
+
+  <div class="parent-container">
+    <div class="snapshot"><img src="snapshotimages/Snapshoot Portfolio (1).png" alt="snapshot"></div>
+    <div class="individual-card">
+      <div><h2>Multi-Post Stories</h2></div>
+      <nav class="nav-elements">
+        <div><a class="current-nav" href="http://">Canopy</a></div>
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">Back End Dev</a></div>
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">2015</a></div>
+      </nav>
```

Comment:
- [ ] The styling of your card is good but this list of information about the project is not a `nav` item. There are no elements here that should redirect the user to another page so please change the `nav` and `a` tags and use `ul` with `li` for the items inside.